src/idls/dip20.did.ts

```diff
@@ -1,109 +1,84 @@
-/* eslint-disable camelcase */
-/* eslint-disable @typescript-eslint/camelcase */
 export default ({ IDL }) => {
-  const TxReceipt = IDL.Variant({
-    ok: IDL.Nat,
-    err: IDL.Variant({
-      InsufficientAllowance: IDL.Null,
-      InsufficientBalance: IDL.Null,
-    }),
+  const TxError = IDL.Variant({
+    'InsufficientAllowance' : IDL.Null,
+    'InsufficientBalance' : IDL.Null,
+    'ErrorOperationStyle' : IDL.Null,
+    'Unauthorized' : IDL.Null,
+    'LedgerTrap' : IDL.Null,
+    'ErrorTo' : IDL.Null,
+    'Other' : IDL.Null,
+    'BlockUsed' : IDL.Null,
+    'AmountTooSmall' : IDL.Null,
   });
+  const Result = IDL.Variant({ 'Ok' : IDL.Nat, 'Err' : TxError });
   const Metadata = IDL.Record({
-    fee: IDL.Nat,
-    decimals: IDL.Nat8,
-    owner: IDL.Principal,
-    logo: IDL.Text,
-    name: IDL.Text,
-    totalSupply: IDL.Nat,
-    symbol: IDL.Text,
+    'fee' : IDL.Nat,
+    'deploy_time' : IDL.Nat64,
+    'decimals' : IDL.Nat8,
+    'fee_to' : IDL.Principal,
+    'owner' : IDL.Principal,
+    'history_size' : IDL.Nat64,
```

Comment:
Sadly this is a mistake, there is no history_size in the standard, but the DFinance guys put it into the implementaiton. We can leave this here, now, but future changes will be coming.